### ISSUE TITLE
(Proposal) Rename Preferences > Interface > Icon Size as "Level Strip Icon Size"

### ISSUE THREAD

**Issue by MrBlockCat** [opentoonz/opentoonz#3136]
Icon Size, from an outside perspective, sounds like all the icons in OpenToonz should change with it. However, what it changes is the icons in the Level Strip. To make this clearer to beginners, the "Icon Size" setting should probably be renamed to "Level Strip Icon Size"

**Comment by RodneyBaker** [comment]
As this is a fairly trivial change... almost equivalent to a typo... it mostly just needs a yes, no or 'table this proposal for now' confirmation.

Recommend those in favor give the proposal a thumbs up, those opposing; thumbs down, and those desiring to table the proposal post their thoughts on why it is best to wait.

In the meantime the proposal has been added to the 2020 UI Scrub project.

**Comment by MrBlockCat** [comment]
If it's decided we should do this, I can pretty easily! It's at https://github.com/opentoonz/opentoonz/blob/2d54639676700d531f2a5f6188f53bce9cd84c4c/toonz/sources/toonz/preferencespopup.cpp, aka opentoonz/toonz/sources/toonz/preferencespopup.cpp, at line 962!

**Comment by gab3d** [comment]
I agree with the proposal.

Also, I've always wondered why the more common term "Thumbnail" is not used in OT for this.

At least to me, an "Icon" refers more to a graphic element that is designed to represent some kind of specific concept, and as a UI item, one the user can make click on to perform a specific predefined operation.
"Thumbnail", on the other hand, is more commonly used as miniature representations of images or other graphic media the user works with, mainly for the purpose of quick identification.

So, why not take the opportunity and replace the term?

**Comment by MrBlockCat** [comment]
@gab3d I 100% agree! I figured Icon would be less controversial since it's more similar to what was there before, but if others agree Thumbnail is a better term I can make a new PR with that change.

@RodneyBaker Is it best if I also make another proposal to change "Icon" to "Thumbnail" before starting the PR?

**Comment by RodneyBaker** [comment]
I believe you can just go direct with the PR... referencing this issue.
Then we'll give the PR a little time to allow feedback and press forward as required.

As for 'thumbnail' versus 'icon' I can see pros and cons to both but agree that using the drawing/animation term 'thumbnail' would be useful.  Of course, I'm curious how 'thumbnail' might translate in some languages.  Perhaps it might translate as 'icon'.  ;)

**Comment by MrBlockCat** [comment]
Hmm, good idea with Frame. I'm inclined to go with that over Thumbnail, both for accuracy and because Thumbnail might make the phrase too long in the UI! Thoughts on "Level Strip Frame Size*:" @gab3d?

**Comment by konero** [comment]
Go with thumbnail, it's the standard term for this.

**Comment by gab3d** [comment]
@RodneyBaker  @MrBlockCat 
> While I like the term 'thumbnail' it seems that for the Level Strip the more proper term would be 'frame'. 

well, as I see it, "Frame" would sound to me as another possible way to refer to a "Drawing" (i.e. the full image contained in a level frame), on the other hand "Thumbnail" points more clearly to being a miniature representation of something not seen in "its full form". 

also, for example we could say things like "the Thumbnail of the Drawing" and it will be clear what we're talking about; while the phrase "the Frame of the Drawing" will not be immediately understandable without a proper context: i.e. a frame number, one of the miniature frames used in the Level Editor, etc. 

also concepts like "Frame Size" would become ambiguous, and some user may think we're referring to the Drawing size, and not the miniature size.
well, you get the idea 😸

**Comment by RodneyBaker** [comment]
@MrBlockCat Just a check to see if you are still interested in implementing this change.
Or should the proposal be tabled?

**Comment by MrBlockCat** [comment]
It ended up being more complicated due to translation stuff than I'd initially thought, so I can't do it personally. If someone else that understands all of that wants to make the change, that would be awesome! Otherwise, I suppose the proposal should be tabled. Sorry about that!
